extensions/ql-vscode/src/query-history.ts

```diff
@@ -571,12 +599,16 @@ export class QueryHistoryManager extends DisposableObject {
   }
 
   async handleCompareWith(
-    singleItem: FullQueryInfo,
-    multiSelect: FullQueryInfo[]
+    singleItem: LocalQueryInfo,
+    multiSelect: LocalQueryInfo[]
```

Comment:
I think I need to do this:

```suggestion
    singleItem: QueryHistoryInfo,
    multiSelect: QueryHistoryInfo[]
```